webapp_frontend/src/background/api/api.ts

```diff
@@ -1,16 +1,25 @@
-import Axios, {} from "axios";
-import {constants} from "../constants";
+import Axios, { } from "axios";
+import { constants } from "../constants";
 
 export const hostname: string = constants.url.API_URL;
 
 export const userPath: string = '/v1/users';
 
 export const filesystemPath: string = '/v1/filesystem';
 
+enum DataIntegrity {
+    STABLE = "systemHealth/traffic_green.png",
+    POSSIBLE_RISK = "systemHealth/traffic_yellow.png",
+    UNSTABLE = "systemHealth/traffic_red.png"
+}
```

Comment:
These images still need to be created.

Reply:
@Gimleux

Reply:
https://material.io/resources/icons/?icon=traffic&style=baseline